### DISCUSSION TITLE
Frontend code for Admin Panel

### DISCUSSION THREAD

**Discussion by detronetdip** [detronetdip/E-commerce#4] [Announcements]
Currently the frontend folder only contains the the frontend code for seller, deliveryboy and user, within few days  the code for admin panel will be added.

**Comment by detronetdip** [comment]
Few code has been added. Rest of them will be added soon.

**Comment by detronetdip** [comment]
All files for `Admin Panel` has been added.

**Reply by 5cr33ch** [reply]
Hi!  What's the url path to access that admin panel?  also, since the password is hashed what is the existing password for admin@gmail.com ?  I can't seem to find the information anywhere.  thanks!

**Reply by detronetdip** [reply]
@5cr33ch  add `/Admin` after home page URL for accessing the admin panel and the password is `admin`
